### DISCUSSION TITLE
Paying for a purchase in installments

### DISCUSSION THREAD

**Discussion by pippo73** [ellite/Wallos#491] [Ideas]
Hi all, 
I've discovered your app, and I find your project interesting.
I would like to know if it is possible to add also a payment, which will be in several times, but it is not a subscription.
For example, I can buy something, choose to pay it in the amount in 3 times, and I would like to trace it.
Is it possible?

**Comment by gbcatrinoiu** [comment]
This is something that I was looking for as well.

**Comment by DamagedDingo** [comment]
Me too.. AfterPay for example.
Also, I get paid fortnightly so somethings like yearly Rego, or monthly phone bills i sit aside in installments. Might be outside the scope of this and require a fork.
